pkg/proc/bininfo.go

```diff
@@ -365,8 +366,19 @@ func FindFunctionLocation(p Process, funcName string, lineOffset int) ([]uint64,
 // If sameline is set FirstPCAfterPrologue will always return an
 // address associated with the same line as fn.Entry.
 func FirstPCAfterPrologue(p Process, fn *Function, sameline bool) (uint64, error) {
+	// The DWARF prologue_end statement is incorrect in ppc64le and
```

Comment:
I'm very skeptical that this code can be correct. How does this skip the prologue? Not skipping the prologue will cause a number of problems. If the prologue_end position is wrong then:

1. there should be a PR to fix it against cmd/link
2. the disassembly method should be used